client/src/components/UserPage.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import MessageList from './MessageList.jsx'
+
+const UserPage = () => (
```

Comment:
Add 'props' as a parameter to UserPage in case we want to send data down the line  (const UserPage = (props) ();

Reply:
let's add it in once we've identified which data to pull in from parent component?